test/integration/access/controls/base_access.rb

```diff
@@ -1,8 +1,6 @@
 control 'postgresl-local-access' do
```

Comment:
Just because i wanna tell something. Better put postgres**q**l instead of postgresl  in the name of the control